### DISCUSSION TITLE
How to show the sidebar?

### DISCUSSION THREAD

**Discussion by LeonardoGentile** [alefragnani/vscode-bookmarks#706] [Q&A] [answered]
I've installed the extension but I can't find a way to enable and view the bookmark sidebar.
If I right click on the sidebar it doesn't appear to be an option related to the bookmarks extension.

On the command palette if I bring up "bookmarks" there is nothing there to show/hide the sidebar. 

I've uninstalled the extension/closed vscode (instead of reloading) and reinstalled it again as suggested in here https://github.com/alefragnani/vscode-bookmarks/discussions/449 but I had no luck, the sidebar is not there and there is nothing suggesting how to enable it 🤷‍♂️

**Comment by alefragnani** [comment, marked as answer]
Hi @LeonardoGentile ,

You could try using the `Bookmarks: Focus on Explorer View` command, which is a built in command created by VS Code when extensions provides those views. This command should bring you the Bookmarks view, wherever it is.

Another command you could use tis `Views: Reset View Locations`, but use carefully, if you customize your workspace too much, because it will _reset_ the customizations and arranges you made on views, moving them to its original location.

Hope this helps

**Comment by LeonardoGentile** [comment]
> You could try using the Bookmarks: Focus on Explorer View command, which is a built in command created by VS Code when extensions provides those views. 

I don't have this command in my vscode 🤔

> Views: Reset View Locations 

This helped, thanks!
